vllm/engine/multiprocessing/client.py

```diff
@@ -685,23 +686,37 @@ async def reset_prefix_cache(self) -> None:
             request=RPCResetPrefixCacheRequest.RESET_PREFIX_CACHE,
             socket=self.input_socket)
 
-    async def add_lora(self, lora_request: LoRARequest) -> None:
-        """Load a new LoRA adapter into the engine for future requests."""
-        # Uses the same I/O as generate requests
-        request = RPCLoadAdapterRequest(lora_request)
-
+    async def send_request_and_get_response(self, request: Any) -> Any:
```

Comment:
struggling on how to get the response from the engine, @robertgshaw2-redhat can you please help 🥺